mitodl/open-discussions
# Added tasks and commands to fetch youtube videos

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #2335 

#### What's this PR do?
- Adds cron task for syncing 
- Adds a management command that supports delete and fetch operations

#### How should this be manually tested?
- Run the task, it should immediately complete
- Run the command, a few examples:
  - `./manage.py backpopulate_youtube_data delete` - removes all `Video` records for youtube
  - `./manage.py backpopulate_youtube_data fetch` - fetch all youtube data (same as the cron task)
  - Run the above fetch commands also specifying `--channel-id ID`, where `ID` is one listed here: https://github.com/mitodl/open-video-data/blob/master/youtube.yaml
